index.html

```diff
@@ -36,6 +36,191 @@ <h2 class="lets-connect">LET'S CONNECT</h2>
+                    <li class="back">2015</li>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>   
+        </div>
+
+        <div class="works works4">
+            <div class="w_ctn">
+                <img src="images/Snapshoot Portfolio (3).svg" alt="" class="image4">
+                <h2 class="header">Multi-Post</h2>
+
+                <ul class="description">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">Back End Dev</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">2018</li>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+            <section class="illustration">
+            </section>
+        </div>
+
+        
+    </section>
+
+
+    <section class="about-myself" id="about-myself">
+        <div class="myself">
+            <div>
+                <h2 class="about">About Myself</h2>
+                <p class="experience">Hello I'm a software developer! 
+                        I can help you build a product , feature or website Look 
+                        through some of my work and experience! If you like what you 
+                        see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <h3 class="connect-1">LET'S CONNECT</h3>
+
+                <ul class="icons-1">
+                    <li><img src="images/Vector (4).svg" class="icons-2"></li>
+                    <li><img src="images/Group.svg" class="icons-2"></li>
+                    <li><img src="images/Vector (5).svg" class="icons-2"></li>
+                    <li><img src="images/Vector.svg" class="icons-2"></li>
+                    <li><img src="images/Vector (3).svg" class="icons-2"></li>
+                </ul>
```

Comment:
- [ ] It is required that the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Kindly use <a> tag to add links to your social icons.

- [ ] Kindly reduce the left-margin of all the icons in order to perfectly match the figma design

### Your Design
![your-about](https://github.com/MarkYegon7/Mark-Portfolio/assets/26597432/f315542d-3b62-4161-ae75-650a48f96a0a)

### Figma Design
![temp-about](https://github.com/MarkYegon7/Mark-Portfolio/assets/26597432/46382579-c80a-474c-8e27-8fc8029d7873)

Reply:
Implemented the said changes